gateway/src/main/kotlin/com/example/gateway/mapper/OrderProtoMapper.kt

```diff
@@ -197,4 +144,58 @@ object OrderProtoMapper {
             it.index = index
         }.build()
     }
+
+    @SuppressWarnings("ThrowsCount")
+    private fun CreateOrderResponse.failureCase(): Nothing {
+        when (failure.errorCase!!) {
```

Comment:
```suggestion
        throw when (failure.errorCase!!) {
```
and no need to suppress ThrowsCount

or refactor function to return Exception and throw it in call place